data/brands/amenity/payment_terminal.json

```diff
@@ -4,6 +4,18 @@
     "exclude": {"generic": ["^payment terminal$"]}
   },
   "items": [
+    {
+      "displayName": "BottleDrop Plus",
```

Comment:
Are these all actually BottleDrop Plus branded. I know there's a kiosk in some Wholefoods and Trader Joes locations, but they're not in the Bottle Drop Plus 20% program.

Reply:
You're right, some of the kiosks aren't BottleDrop Plus branded. I'll add another entry with just 'BottleDrop'.